iina/Aspect.swift

```diff
@@ -12,24 +12,6 @@ class Aspect: NSObject {
 
   private var size: NSSize!
 
-  var width: CGFloat {
-    get {
-      return size.width
-    }
-    set {
-      size.width = newValue
-    }
-  }
-
-  var height: CGFloat {
-    get {
-      return size.height
-    }
-    set {
-      size.height = newValue
-    }
-  }
-
```

Comment:
We should probably keep this even if it's unused

Reply:
@saagarjha fixed: https://github.com/iina/iina/pull/4425/commits/1d610bdd56709994411939c29b895d90177539c1